README.md

```diff
@@ -85,6 +82,9 @@ kubectl get -n astria-dev-cluster nodes
 kubectl get --all-namespaces pods
 kubectl get -n astria-dev-cluster pods
 
+# list all containers within a pod
+kubectl get -n astria-dev-cluster POD_NAME -o jsonpath='{.spec.containers[*].name}'
```

Comment:
This never worked for me. I think this is better:
```
kubectl get -n astria-dev-cluster deployment sequencer -o jsonpath='{.spec.template.spec.containers[*].name}'
```

Reply:
Ah, there's a missing `pod` in the command you have. This works
```
kubectl get -n astria-dev-cluster pod sequencer-57b8685c55-4l556 -o jsonpath='{.spec.containers[*].name}'
```

It still requires you to get the pod name first though.